node/lib/util/submodule_config_util.js

```diff
@@ -71,6 +71,34 @@ const CONFIG_FILE_NAME = "config";
 exports.modulesFileName = ".gitmodules";
 
 /**
+ * Return the relative path from the working directory of a submodule having
+ * the specified `subName` to its .git directory.
+ * @param {String} subName
+ */
+exports.computeRelativeGitDir = function (subName) {
```

Comment:
I'm not sure what this concern has to do with `computeRelativeGitDir`, which is just how to point the checked out directory back to the .git directory.